openpype/hosts/maya/plugins/load/load_audio.py

```diff
@@ -59,8 +59,24 @@ def update(self, container, representation):
         assert audio_nodes is not None, "Audio node not found."
         audio_node = audio_nodes[0]
 
+        current_sound = cmds.timeControl(
+            mel.eval("$gPlayBackSlider=$gPlayBackSlider"),
+            query=True,
+            sound=True
+        )
+        activate_sound = current_sound == audio_node
+
         path = get_representation_path(representation)
         cmds.setAttr("{}.filename".format(audio_node), path, type="string")
+
+        if activate_sound:
+            cmds.timeControl(
```

Comment:
Should we add a comment why we're doing this?
```suggestion
            # maya by default deactivates it from timeline on file change
            cmds.timeControl(
```